### ISSUE TITLE
state eclipse-bookmarks fails

### ISSUE THREAD

**Issue by noelmcloughlin** [saltstack-formulas/eclipse-formula#13]
Because parent directory ~/workspace/ does not exist.   Fedora/Ubuntu/Suse

**Comment by noelmcloughlin** [comment]
```
          ID: eclipse-bookmarks
    Function: file.managed
        Name: /home/messi/workspace/salted-bookmarks.xml
      Result: False
     Comment: Parent directory not present
     Started: 19:13:29.933628
    Duration: 2.451 ms
     Changes:   
```

**Comment by noelmcloughlin** [comment]
Fixed by #16
